gradle/src/main/groovy/com/flurry/android/symbols/SymbolUploadConfiguration.groovy

```diff
@@ -12,6 +12,7 @@ class SymbolUploadConfiguration {
     boolean useEnvVar = true
     String configPath = null
     int uploadTimeout = 300
+    boolean ndk
```

Comment:
`boolean ndk = false` for consistency

Reply:
`false` is default type for and `boolean`. And I have removed most other redundant initializations in #13.